api/src/main/java/org/openmrs/module/aijar/tasks/EnableDisablePatientFlags.java

```diff
@@ -0,0 +1,40 @@
+package org.openmrs.module.aijar.tasks;
+
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Disable Patient Flags
+ */
+public class EnableDisablePatientFlags {
```

Comment:
@slubwama No need for this class as a method in the activator should work just fine